specification/cognitiveservices/data-plane/QnAMaker/readme.md

```diff
@@ -0,0 +1,99 @@
+# Cognitive Services QnAMaker SDK
+
+> see https://aka.ms/autorest
+
+Configuration for generating QnAMaker SDK.
+
+The current release is `release_4_0`.
+
+``` yaml
+
+tag: release_4_0
+add-credentials: true
+openapi-type: data-plane
+```
+# Releases
+
+### Release 4.0
+These settings apply only when `--tag=release_4_0` is specified on the command line.
+
+``` yaml $(tag) == 'release_4_0'
+input-file: stable/v4.0/QnAMaker.json
+```
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+``` yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python
+  - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-go
+  - repo: azure-sdk-for-js
```

Comment:
looks like js and node SDKs configuration is missing, if you look at the PR comments, there's no generated SDK PR out of those, please take a look at https://github.com/Azure/azure-rest-api-specs/tree/master/specification/sql/resource-manager to add readme.md files for js and node to configure SDK generation for them.

Reply:
I would recommend to take a look at a PR that got generated for one of the language SDKs (see PR comments), to check that the generated code looks as you'd like it to be.

Reply:
done

Reply:
ruby sdk fails. can you please take a look ?